models/pardot/model.sql

```diff
@@ -81,3 +81,24 @@ create or replace view pardot_analysis.visitoractivity_events_meta as (
   select 18, '', 'reopened opportunity'
 
 );
+
+
+create or replace view {schema}.visitoractivity as (
+  --this table has a bunch of types that really should be event actions but are very poorly formulated.
+  --the custom logic in this view is an attempt to fix that.
+  --not all of the various type / type_name combinations have been accounted for yet; I still need to determine exactly what some of them mean.
+  select
+    -- event_stream interface
+    va.created_at       as "@timestamp",
```

Comment:
```
SQL identifiers and key words must begin with a letter (a-z, but also letters with diacritical marks and non-Latin letters) or an underscore (_). Subsequent characters in an identifier or key word can be letters, underscores, digits (0-9), or dollar signs ($). Note that dollar signs are not allowed in identifiers according to the letter of the SQL standard, so their use might render applications less portable. The SQL standard will not define a key word that contains digits or starts or ends with an underscore, so identifiers of this form are safe against possible conflict with future extensions of the standard.
```

according to the postgres docs, it doesn't sound like `@` is standard or that we can use anything outside of `[a-zA-Z_]` as the first character in a column name. So maybe our best bet is to do `_ac_` or something like that. Open to suggestions 

via http://www.postgresql.org/docs/9.1/static/sql-syntax-lexical.html